docs/tutorials/11_quantum_convolutional_neural_networks.ipynb

```diff
@@ -732,37 +733,33 @@
     "ansatz.compose(pool_layer([0], [1], \"p3\"), list(range(6, 8)), inplace=True)\n",
     "\n",
     "# Combining the feature map and ansatz\n",
-    "circuit = QuantumCircuit(8)\n",
-    "circuit.compose(feature_map, range(8), inplace=True)\n",
-    "circuit.compose(ansatz, range(8), inplace=True)\n",
+    "circuit = QNNCircuit(feature_map=feature_map, ansatz=ansatz)\n",
     "\n",
     "observable = SparsePauliOp.from_list([(\"Z\" + \"I\" * 7, 1)])\n",
     "\n",
     "# we decompose the circuit for the QNN to avoid additional data copying\n",
     "qnn = EstimatorQNN(\n",
-    "    circuit=circuit.decompose(),\n",
+    "    circuit=circuit,\n",
```

Comment:
I am not sure of the statement around decompose and additional data copying - I would check this in regards of performance by adding back decompose and see what happens. It maybe that this decomposition speeds things up around binding parameters - maybe internally some copies are avoided during binding this way. Just so happens that a PR was opened in Qiskit earlier today, that is likely related, to flatten off these blocks in NLocal circuits as it leads to way faster binding. I imagine this is more or less what decompose was used here for.

Reply:
@woodsp-ibm  thanks for your insights!
@adekusar-drl I will try to figure out what caused the long eval times and set back the nbsphinx timeout to its original value :).

Reply:
FYI: related to decompose above (has a plot of times as comparison)

* Qiskit/qiskit-terra/issues/10282

Reply:
A small increase in execution time was observed when decomposing the QNNCircuit (several times - using reps =2,3,4). The respective execution time was still much longer than the original version. This is most definitely linked to the issue @woodsp-ibm referred to.
For now, the QNNCircuit class will be demonstrated in tutorials: 02 & 10.  
Once the issue  [Qiskit/qiskit-terra#10282](https://github.com/Qiskit/qiskit-terra/pull/10284) is closed I will pick up the adjustments for tutorial 05 & 11.

Reply:
From my perspective its curious that something which should be more or less identical to the circuit that was formed separately before here behaves differently - is this the case elsewhere too, ie that the performance of using QNNCircuit elsewhere is worse too, just that here it got noticed. The only thing that occurs to me is that the combined circuit before was a plain QuantumCircuit that contained the composed BlueprintCircuit's (i.e feature map and ansatz) whereas QNNCircuit is a BlueprintCircuit itself. Whatever the difference it maybe that the changes going on in Qiskit improve things in any regards making the decompose hopefully unneeded and maybe even making things more comparable. Anyway I just figured I'd mention this since ideally things ought to be better understood so as QNNCircuits use as a convenience is not adversely impacting performance more generally but probably that's best left to after whatever changes Qiskit brings to the parameter binding.